samples/dnn/dnn_model_runner/dnn_conversion/paddlepaddle/README.md

```diff
@@ -0,0 +1,80 @@
+# Run PaddlePaddle model by OpenCV
+
+This tutorial gives 2 demos show how to run PaddlePaddle model by opencv
+
+## Environment Setup
+
+```shell
+pip install paddlepaddle-gpu
+pip install paddlehub
+pip install paddle2onnx
+```
+
+## 1. Run PaddlePaddle ResNet50 by OpenCV
+
+### Run PaddlePaddle model demo
+
+Run the example code as below,
+
+```shell
+python paddle_resnet50.py
+```
+
+there are 3 part of this execution
+
+* 1. Export PaddlePaddle ResNet50 model to onnx format;
+* 2. Use `cv2.dnn.readNetFromONNX` load model file;
+* 3. Preprocess image file and do inference.
```

Comment:
3. Preprocess image file and do the inference.

Reply:
Done.